test/karma.conf.js

```diff
@@ -0,0 +1,70 @@
+// Karma configuration
+// Generated on Fri Mar 20 2015 17:21:46 GMT+0000 (GMT)
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
```

Comment:
If you set your basePath to "../", then you won't need the "../"s in the list of files to load below because it will always start looking from that position.